pinc/LPage.inc

```diff
@@ -320,15 +320,15 @@ class LPage
         Page_markAsBad( $this->projectid, $this->imagefile, $this->round, $user, $reason );
 
         // Advise PM that the page has been marked bad.
-        $body_blurb =
-"Page {$this->imagefile} of this project has been marked bad due to {$PAGE_BADNESS_REASONS[$reason]}.
-Please visit
-    $code_url/tools/project_manager/handle_bad_page.php?projectid={$this->projectid}&image={$this->imagefile}
-to make any needed changes and make the page available for proofreading again.
-Until this report has been resolved, the project will not be able to leave
-the current round. If 10 pages are marked bad by at least 3 different users,
-the project will automatically be made unavailable.";
-        maybe_mail_project_manager($this->projectid, $body_blurb, "$site_abbreviation: Bad Page");
+        $body_blurb_messages[] = sprintf(_("Page %s of this project has been marked bad due to %s."), $this->imagefile, $PAGE_BADNESS_REASONS[$reason]);
```

Comment:
When there are two strings in a translated statement, we need to number them in case the translation swaps their position. See https://www.pgdp.net/wiki/DP_Code_Best_Practices#Using_sprintf.28.29_with_gettext for an example.